onert-micro/luci-interpreter/src/kernels/Add.test.cpp

```diff
@@ -164,3 +164,79 @@ TEST_F(AddTest, No_quant_params_NEG)
 
 } // namespace
 } // namespace luci_interpreter
+
+#include "PALAdd.h"
+
+#include "BinaryOpCommon.h"
+
+#include <array>
+#include <numeric>
+
+namespace luci_interpreter
+{
+namespace
+{
+
+class PALAddTest : public ::testing::Test
+{
+  // Do nothing
+};
+
+TEST_F(PALAddTest, Float_P)
```

Comment:
I couldn't add negative tests because there is no case that kernels in PAL throws exception.